tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -0,0 +1,150 @@
+|---|---|
+| AuditEventSignIn | sign in |
+| AuditEventSignOut | sign out |
+| AuditEventUpdateUserProfile | update user's first name and last name |
+| AuditEventUpdateMFA | enable or disable MFA |
+| AuditEventCreateProject | create a new project |
+| AuditEventUpdateProject | update the project name |
+| AuditEventDeleteProject | delete a project |
+| AuditEventInviteUserIntoProject | invite a user into project |
+| AuditEventDeleteProjectUser | delete a project user |
+| AuditEventUpdateOrg | update organization name and time zone |
+| AuditEventCreateIntegration | create an integration |
+| AuditEventDeleteIntegration | delete an integration |
+| AuditEventListOrgUsers | list users in organization |
+| AuditEventListProjectUsers | list users in a project |
+| AuditEventAddNewPaymentMethod | add a new credit card |
+| AuditEventUpdatePaymentMethod | update credit card info |
+| AuditEventDeletePaymentMethod | delete a credit card |
+| AuditEventCreateAWSVpcPeering | create an aws vpc peering |
+| AuditEventCreateGCPVpcPeering | create a gcp vpc peering | 
+| AuditEventListAWSVpcPeering | list all aws vpc peering in the project |
+| AuditEventListGCPVpcPeering | list all gcp vpc peering in the project |
+| AuditEventDeleteAWSVpcPeering | delete an aws vpc peering |
+| AuditEventDeleteGCPVpcPeering | delete a gcp vpc peering |
+| AuditEventGetProjectTrafficFilter | get traffic filter list of the project |
+| AuditEventUpdateProjectTrafficFilter | update traffic filter list of the project |
+| AuditEventGetTrafficFilter | get traffic filter list of the cluster |
+| AuditEventUpdateTrafficFilter | update traffic filter list of the cluster |
+| AuditEventCreateProjectCIDR | create a new project CIDR |
+| AuditEventGetProjectCIDR | list CIDR in the region |
+| AuditEventGetProjectRegionCIDR | list all CIDR in the project |
+| AuditEventDeleteBackupInRecycleBin | delete backups of deleted clusters in recycle bin |
+| AuditEventChangeClusterRootPassword | reset the root password of cluster |
+| AuditEventCreateImportTask | create an import task |
+| AuditEventCancleImportTask | cancel an import task |
+| AuditEventExitImportTask | exit an import task |
+| AuditEventCreateCluster | create a cluster |
+| AuditEventDeleteCluster | delete a cluster |
+| AuditEventScaleCluster | scale a cluster |
+| AuditEventCreateBackup | make a backup |
+| AuditEventDeleteBackup | delete a backup |
+| AuditEventRestoreBackup | restore from a backup |
+| AuditEventUpdateAuditLogStatus | enable / disable audit logging |
+| AuditEventCreateAuditLogAccessRecord | add audit log filter conditions |
+| AuditEventDeleteAuditLogAccessRecord | delete audit log filter conditions |
+| AuditEventUpdateUserRole | modify user roles |
+
+> **Note:**
+>
+> - Closing the Console audit log feature is not supported;
+> - Setting filter rules is not supported.
+
+## Audit log storage policy
+- Audit log information is currently only supported to be stored in AWS ES;
```

Comment:
If users cannot access the log independently, they cannot do anything about it, including deciding where the logs can be stored.
```suggestion

- The audit log information is stored in AWS ES.
```